src/content/release-notes.mdx

```diff
@@ -25,7 +25,7 @@ This version is compatible with Kubernetes versions 1.27 to 1.30
 ### Breaking Changes
 
 - **Hostname Length Limit**: Deploys now fail if a service hostname exceeds 63 characters, showing an error message. Previously, the dev environment deployed successfully even if the endpoint didn’t work. This may affect environments that deployed without issues before
-- **Helm Release Name Limit**: Helm release names are now limited in length. Use `defaultBackend.nameOverride` to shorten the default backend service name if needed. If changing this during an upgrade, [follow this guide as it may impact the installation](self-hosted/helm-configuration.mdx#manual-migration-steps-when-renaming-the-defaultbackend-service)
+- **Helm Release Name Limit**: Helm release names are now limited in length. Use `defaultBackend.nameOverride` to shorten the default backend service name if needed. If changing this during an upgrade, [follow this guide as it may impact the installation](/docs/1.26/self-hosted/helm-configuration.mdx#manual-migration-steps-when-renaming-the-defaultbackend-service)
```

Comment:
this link does not exist yet, but it should exist once we release 1.27 and 1.26 becomes versioned.

Reply:
@codyjlandstrom do you know a better way to link to that paragraph that will only exist in 1.26?

Reply:
I'm not quite sure I follow. We want to add the link to 1.26, but only after the release of 1.27?